### PR TITLE
Expanded view mode, Edit Title and Description

### PR DESCRIPTION
**Pull request by hector0513**
image

### CODE REVIEW

**Comment by alex1812r** [comment]
image

**Comment by Connoropolous** [comment]
I will try to look at this as soon as possible

File: src/components/ExpandedViewMode/ExpandedViewModeContent/ExpandedViewModeContent.js [hunk 1]
```diff
@@ -1,16 +1,86 @@
...
+    setEditDescription(false)
+  }
+
+  const handlekeyPress = ({ key }) => {
+    if (key === 'Enter') {
+      updateContent()
+    }
+  }
+
+  const handleOnChange = ({ target }) => {
```
Comment (Connoropolous) [inline src/components/ExpandedViewMode/ExpandedViewModeContent/ExpandedViewModeContent.js]: Maybe just create two separate event handlers?

File: src/holochainConfig.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export const DEVELOPMENT_INSTANCE_NAME = 'acorn'
 export const ZOME_NAME = 'holo_acorn'
 export const DEFAULT_HOLOCHAIN_PORT = '8888'
-export const DEFAULT_HOLOCHAIN_HOST = '127.0.0.1'
+export const DEFAULT_HOLOCHAIN_HOST = '192.168.1.63'
```
Comment (Connoropolous) [inline src/holochainConfig.js]: Revert this back before merging.

**Review by Connoropolous — CHANGES_REQUESTED** [review]
This is a very small 'request changes'. It looks and works very well

File: src/holochainConfig.js (hunk 2 above)
Comment (hector0513) [reply src/holochainConfig.js]: done

File: src/components/ExpandedViewMode/ExpandedViewModeContent/ExpandedViewModeContent.js (hunk 1 above)
Comment (hector0513) [reply src/components/ExpandedViewMode/ExpandedViewModeContent/ExpandedViewModeContent.js]: done

File: src/components/ExpandedViewMode/ExpandedViewModeContent/ExpandedViewModeContent.js [hunk 3]
```diff
@@ -1,16 +1,89 @@
...
+    setEditDescription(false)
+  }
+
+  const handlekeyPress = ({ key }) => {
+    if (key === 'Enter') {
+      updateContent()
+    }
+  }
+
+  const handleOnChangeTittle = ({ target }) => {
```
Comment (Connoropolous) [inline src/components/ExpandedViewMode/ExpandedViewModeContent/ExpandedViewModeContent.js]:
```suggestion
  const handleOnChangeTitle = ({ target }) => {
```

File: src/components/ExpandedViewMode/ExpandedViewModeContent/ExpandedViewModeContent.js [hunk 4]
```diff
@@ -1,16 +1,89 @@
...
+        onClick={() => {
+          setEditTitle(true)
+        }}>
+        {editTitle ? (
+          <input
+            type='text'
+            defaultValue={goal.content}
+            onBlur={updateContent}
+            onChange={handleOnChange}
+            onKeyPress={handleOnChangeTittle}
```
Comment (Connoropolous) [inline src/components/ExpandedViewMode/ExpandedViewModeContent/ExpandedViewModeContent.js:58]:
```suggestion
            onKeyPress={handleOnChangeTitle}
```

**Review by Connoropolous — APPROVED** [review]
once you merge those tiny spelling fixes, merge it :)

**Pull request merged**